### PR TITLE
bitget: add spot plan order

### PR DESCRIPTION
**Pull request by sc0Vu**
Bitget had added apis for spot plan order. In this PR I added these apis (https://bitgetlimited.github.io/apidoc/en/spot/#place-plan-order).

SPOT:
```BASH
$ node examples/js/cli bitget createOrder ADA/USDT limit buy 5000 0.001 '{"triggerPrice":"0.002"}'
$ node examples/js/cli bitget fetchOpenOrders ADA/USDT 0 100 '{"stop":true}'
$ node examples/js/cli bitget fetchOpenOrders ADA/USDT 0 100 '{"stop":true}'
$ node examples/js/cli bitget editOrder '"xxxxxxxx"' ADA/USDT limit buy 5000 0.0013 '{"triggerPrice":"0.0013"}'
$ node examples/js/cli bitget fetchClosedOrders ADA/USDT 0 100 '{"stop":true}'
```

SWAP
```BASH
$ node examples/js/cli bitget createOrder ADA/USDT:USDT limit buy 5000 0.001 '{"triggerPrice":"0.002"}'
$ node examples/js/cli bitget fetchOpenOrders ADA/USDT:USDT 0 100 '{"stop":true}'
$ node examples/js/cli bitget fetchOpenOrders ADA/USDT:USDT 0 100 '{"stop":true}'
$ node examples/js/cli bitget editOrder '"xxxxxxxx"' ADA/USDT:USDT limit buy 5000 0.0013 '{"triggerPrice":"0.0013"}'
$ node examples/js/cli bitget fetchClosedOrders ADA/USDT:USDT 0 100 '{"stop":true}'
```

TODO:
- [x] add endpoints for spot plan order
- [x] update create order
- [x] update fetch open orders
- [x] update cancel order
- [x] add edit order (only plan order).

### CODE REVIEW

File: js/bitget.js [hunk 1]
```diff
@@ -176,6 +177,11 @@ module.exports = class bitget extends Exchange {
                             'wallet/transfer': 4,
                             'wallet/withdrawal': 4,
                             'wallet/subTransfer': 10,
+                            'plan/placePlan': 1,
+                            'plan/modifyPlan': 1,
+                            'plan/cancelPlan': 1,
+                            'plan/currentPlan': 1,
+                            'plan/historyPlan': 1,
```
Comment (carlosmiei) [inline js/bitget.js:184]: aren't we missing `fetchClosedOrders` ? 
https://bitgetlimited.github.io/apidoc/en/spot/#get-history-plan-orders

File: js/bitget.js [hunk 2]
```diff
@@ -2299,6 +2307,9 @@ module.exports = class bitget extends Exchange {
             if (isStopOrder) {
                 throw new InvalidOrder (this.id + ' createOrder() does not support stop orders on spot markets, only swap markets');
```
Comment (carlosmiei) [inline js/bitget.js:2308]: hum how would users place spot stopOrders now with this if condition here?

**Comment by carlosmiei** [comment]
@sc0Vu Please attach some test logs (in python preferably) to this PR just to make sure everything is working properly

File: js/bitget.js (hunk 2 above)
Comment (sc0Vu) [reply js/bitget.js:2308]: They can only place trigger order in the bitget, there is no SL/TP order for spot.

File: js/bitget.js (hunk 1 above)
Comment (sc0Vu) [reply js/bitget.js:184]: @carlosmiei I updated fetchClosedOrders, please take a look when you have time.

File: js/bitget.js (hunk 2 above)
Comment (carlosmiei) [reply js/bitget.js:2311]: @sc0Vu ok so the name is the confusing part here, we usually call `stopOrders` `triggerOrders and vice-versa

**Comment by carlosmiei** [comment]
LGTM ✅ 
```
p bitget createOrder "LTC/USDT" market buy 1 5                       
Python v3.10.8
CCXT v2.5.80
bitget.createOrder(LTC/USDT,market,buy,1,5)
{'amount': None,
 'average': None,
 'clientOrderId': '2edcd5c7-1aef-4808-8ad6-a7ec1ffac5a1',
 'cost': None,
 'datetime': None,
 'fee': None,
 'fees': [],
 'filled': None,
 'id': '1000000459967467520',
 'info': {'clientOrderId': '2edcd5c7-1aef-4808-8ad6-a7ec1ffac5a1',
          'orderId': '1000000459967467520'},
 'lastTradeTimestamp': None,
 'postOnly': None,
 'price': None,
 'remaining': None,
 'side': None,
 'status': None,
 'stopPrice': None,
 'symbol': None,
 'timeInForce': None,
 'timestamp': None,
 'trades': [],
 'triggerPrice': None,
 'type': None}
```
```
p bitget createOrder "LTC/USDT" limit buy 0.2 25
Python v3.10.8
CCXT v2.5.80
bitget.createOrder(LTC/USDT,limit,buy,0.2,25)
{'amount': None,
 'average': None,
 'clientOrderId': 'dddf2f4a-f72f-4a00-9719-bd17f882b5b1',
 'cost': None,
 'datetime': None,
 'fee': None,
 'fees': [],
 'filled': None,
 'id': '1000000590007668736',
 'info': {'clientOrderId': 'dddf2f4a-f72f-4a00-9719-bd17f882b5b1',
          'orderId': '1000000590007668736'},
 'lastTradeTimestamp': None,
 'postOnly': None,
 'price': None,
 'remaining': None,
 'side': None,
 'status': None,
 'stopPrice': None,
 'symbol': None,
 'timeInForce': None,
 'timestamp': None,
 'trades': [],
 'triggerPrice': None,
 'type': None}
```
```
p bitget createOrder "LTC/USDT" limit buy 0.2 25 '{"stopPrice":26}'     
Python v3.10.8
CCXT v2.5.80
bitget.createOrder(LTC/USDT,limit,buy,0.2,25,{'stopPrice': 26})
{'amount': None,
 'average': None,
 'clientOrderId': '1000001164593926146',
 'cost': None,
 'datetime': None,
 'fee': None,
 'fees': [],
 'filled': None,
 'id': '1000001164610703360',
 'info': {'clientOrderId': '1000001164593926146',
          'orderId': '1000001164610703360'},
 'lastTradeTimestamp': None,
 'postOnly': None,
 'price': None,
 'remaining': None,
 'side': None,
 'status': None,
 'stopPrice': None,
 'symbol': None,
 'timeInForce': None,
 'timestamp': None,
 'trades': [],
 'triggerPrice': None,
 'type': None}
```
```
p bitget createOrder "LTC/USDT:USDT" limit buy 1 25 '{"stopPrice":26}'
Python v3.10.8
CCXT v2.5.80
bitget.createOrder(LTC/USDT:USDT,limit,buy,1,25,{'stopPrice': 26})
{'amount': None,
 'average': None,
 'clientOrderId': '1000001423776747520',
 'cost': None,
 'datetime': None,
 'fee': None,
 'fees': [],
 'filled': None,
 'id': '1000001423776747521',
 'info': {'clientOid': '1000001423776747520', 'orderId': '1000001423776747521'},
 'lastTradeTimestamp': None,
 'postOnly': None,
 'price': None,
 'remaining': None,
 'side': None,
 'status': None,
 'stopPrice': None,
 'symbol': None,
 'timeInForce': None,
 'timestamp': None,
 'trades': [],
 'triggerPrice': None,
 'type': None}
```
```
 p bitget createOrder "LTC/USDT:USDT" market buy 1 25  
Python v3.10.8
CCXT v2.5.80
bitget.createOrder(LTC/USDT:USDT,market,buy,1,25)
{'amount': None,
 'average': None,
 'clientOrderId': '1000001936469692424',
 'cost': None,
 'datetime': None,
 'fee': None,
 'fees': [],
 'filled': None,
 'id': '1000001936427749380',
 'info': {'clientOid': '1000001936469692424', 'orderId': '1000001936427749380'},
 'lastTradeTimestamp': None,
 'postOnly': None,
 'price': None,
 'remaining': None,
 'side': None,
 'status': None,
 'stopPrice': None,
 'symbol': None,
 'timeInForce': None,
 'timestamp': None,
 'trades': [],
 'triggerPrice': None,
 'type': None}
```

```
p bitget createOrder "LTC/USDT:USDT" market buy 1 25 '{"takeProfitPrice":90}'
Python v3.10.8
CCXT v2.5.80
bitget.createOrder(LTC/USDT:USDT,market,buy,1,25,{'takeProfitPrice': 90})
{'amount': None,
 'average': None,
 'clientOrderId': '1000002012283740160',
 'cost': None,
 'datetime': None,
 'fee': None,
 'fees': [],
 'filled': None,
 'id': '1000002012283740161',
 'info': {'clientOid': '1000002012283740160', 'orderId': '1000002012283740161'},
 'lastTradeTimestamp': None,
 'postOnly': None,
 'price': None,
 'remaining': None,
 'side': None,
 'status': None,
 'stopPrice': None,
 'symbol': None,
 'timeInForce': None,
 'timestamp': None,
 'trades': [],
 'triggerPrice': None,
 'type': None}
```

**Review by carlosmiei — APPROVED** [review]

**Pull request merged**